WolvenKit.RED4/Types/ClassesExt/appearanceAppearanceDefinition.cs

```diff
@@ -12,7 +12,9 @@ public partial class appearanceAppearanceDefinition
     [REDProperty(IsIgnored = true)]
     public CArray<entIComponent> Components
     {
-        get => GetPropertyValue<CArray<entIComponent>>();
-        set => SetPropertyValue<CArray<entIComponent>>(value);
+        get => GetPropertyValue<CArray<entIComponent>>()!; // set in PostConstruct, so not nullable
+        set => SetPropertyValue(value);
     }
+
+    partial void PostConstruct() => Components ??= [];
```

Comment:
Why this change? It could have unforeseen consequences

Reply:
Same as above - it's not annotated as nullable, but the object can leave the constructor with the property not having been set.